README.md

```diff
@@ -1 +1,170 @@
+```
+
+### Check
+
+Use the `psa check` subcommand to verify the cryptographic signature over the supplied PSA attestation token as well as checking whether its claim set is well-formed.
+
+To check the PSA attestation token in my.cbor using the public key in es256.json:
+
+
+```shell
+evcli psa check \
+    --token=my.cbor \
+    --key=es256.json
+```
+
+A message will indicate whether the signature has been successfully verified:
+
+```console
+>> "my.cbor" verified
+```
+
+In such case, the claim set is printed to stdout in JSON format:
+
+```json
+>> embedded claims:
+{
+  "profile": "http://arm.com/psa/2.0.0",
+  "partition-id": 1,
+  "security-life-cycle": 12288,
+  "implementation-id": "UFFSU1RVVldQUVJTVFVWV1BRUlNUVVZXUFFSU1RVVlc=",
+  "boot-seed": "3q2+796tvu/erb7v3q2+796tvu/erb7v3q2+796tvu8=",
+  "hardware-version": "1234567890123",
+  "software-components": [
+    {
+      "measurement-type": "BL",
+      "measurement-value": "AAECBAABAgQAAQIEAAECBAABAgQAAQIEAAECBAABAgQ=",
+      "signer-id": "UZIA/1GSAP9RkgD/UZIA/1GSAP9RkgD/UZIA/1GSAP8="
+    },
+    {
+      "measurement-type": "PRoT",
+      "measurement-value": "BQYHCAUGBwgFBgcIBQYHCAUGBwgFBgcIBQYHCAUGBwg=",
+      "signer-id": "UZIA/1GSAP9RkgD/UZIA/1GSAP9RkgD/UZIA/1GSAP8="
+    }
+  ],
+  "nonce": "AAECAwABAgMAAQIDAAECAwABAgMAAQIDAAECAwABAgM=",
+  "instance-id": "AaChoqOgoaKjoKGio6ChoqOgoaKjoKGio6ChoqOgoaKj",
+  "verification-service-indicator": "https://psa-verifier.org",
+  "_partition-id-desc": "spe",
+  "_security-lifecycle-desc": "secured"
+}
+```
+
+The claim set can also be saved to a file using the `--claims` switch (abbrev.  `-c`), as in:
+
+```shell
+evcli psa check \
+    --token=my.cbor \
+    --key=es256.json \
+    --claims=output-claims.json
+```
+
+### Verify
+
+The `psa verify-as` subcommand allows you to interact with the Veraison Verifier (or another Attestation Verifier implementing the Veraison [challenge-response API](https://github.com/veraison/docs/tree/main/api/challenge-response)).
+
+There are two modes of operation corresponding to the emulated roles: [Attester](#attester) or [Relying Party](#relying-party).
+
+#### Attester
+
+The `attester` subcommand implements the "attester mode" of a challenge-response interaction, where the verifier is the protocol challenger.  Therefore, the nonce is provided by the Veraison API server and the PSA attestation token needs to be created on the fly based on the attester's claims and signing IAK.
+
+```shell
+evcli psa verify-as attester \
+    --api-server=https://veraison.example/challenge-response/v1 \
+    --claims=psa-claims-profile-2-without-nonce.json \
+    --key=es256.json
+```
+
+Note that the supplied claims file must not include a nonce claim.
+
+By default, the command will request 48 bytes nonce from the server.  If needed, a different value can be requested using the `--nonce-size` (abbrev. `-n`) switch.  Available nonce sizes are 32, 48 or 64 bytes, as per PSA attestation token specification.
```

Comment:
A link to PSA attestation token specification will certainly help here.

Reply:
ok